lib/Pod/To/Text.pm6

```diff
@@ -33,14 +33,32 @@ sub pod2text($pod) is export {
 
 sub heading2text($pod) {
     given $pod.level {
-        when 1  {          pod2text($pod.contents)  }
-        when 2  { '  '   ~ pod2text($pod.contents)  }
+        my $content = $pod.contents.first.contents.grep( * !~~ "").first;
```

Comment:
Not sure why you are using `.grep` **and** `.first`, you only need the `.first`

Using only `.first` and not `.grep.first` makes it 2x faster. And then we get more speed up doing this:
```perl6
my $content = $pod.contents.first.contents.first( *.Bool);
```
Which will catch empty strings as well as undefined strings.